### PR TITLE
Handle issues due to bugs in backports of `responses`

### PR DESCRIPTION
**Pull request by crazyscientist**

### CODE REVIEW

File: osctiny/tests/test_utils.py [hunk 1]
```diff
@@ -565,8 +565,11 @@ def test_get_http_error_details__bad_response(self):
                 self.fail("No exception was raised")
 
         with self.subTest("Request count"):
-            if rsp_mock is not None:
-                # In the last version of `responses` supporting Python 3.6, `rsp_mock` is None
+            # In the last version of `responses` officially supporting Python 3.6, `rsp_mock` is
+            # None.
+            # Also, some distros (e.g. openSUSE Leap 15.6) have backported newer versions of
+            # `responses` for Python 3.6.
+            if rsp_mock is not None and sys.version_info >= (3, 6):
```
[the remark this answers is not in the document]
Comment (crazyscientist) [reply osctiny/tests/test_utils.py]: Right. Changed it.

[the remark this answers is not in the document]
Comment (frozenIceage) [reply osctiny/tests/test_utils.py]: Sorry, on my Leap 15.6 container ` > (3, 6)` wouldn't work:
```
>>> sys.version_info > (3, 6)
True
>>> sys.version_info
sys.version_info(major=3, minor=6, micro=15, releaselevel='final', serial=0)
```

[the remark this answers is not in the document]
Comment (crazyscientist) [reply osctiny/tests/test_utils.py]: Right, please see new commit.

[the remark this answers is not in the document]
Comment (frozenIceage) [reply osctiny/tests/test_utils.py]: Thank you!

**Pull request merged**